### PR TITLE
Add create scud item to certs web

### PR DESCRIPTION
**Pull request by ethatcher83**
Bump sdk version
Add post for scud item
Add create scud
Add scud create route
Add unit and integration test

Resolves: GCI-1398

### CODE REVIEW

File: test/client/api.client.unit.test.ts [hunk 1]
```diff
@@ -178,6 +180,50 @@ const dummyCertifiedCopyItemSDKResponse: Resource<CertifiedCopyItem> = {
...
 };
 
+const dummyScudItemSDKResponse: Resource<ScudItem> = {
+    httpStatusCode: 200,
+    resource: {
+        companyName: "Company Name",
+        companyNumber: "00000000",
+        customerReference: "1133XR",
+        description: "certificate",
+        descriptionIdentifier: "certificate",
```
Comment (nabeelamirch) [inline test/client/api.client.unit.test.ts:190]: should this be scud or something?

File: test/client/api.client.unit.test.ts [hunk 2]
```diff
@@ -178,6 +180,50 @@ const dummyCertifiedCopyItemSDKResponse: Resource<CertifiedCopyItem> = {
...
+        descriptionValues: {
+            item: "certificate"
+        },
+        etag: "33a64df551425fcc55e4d42a148795d9f25f89d4",
+        id: "CHS00000000000000004",
```
Comment (nabeelamirch) [inline test/client/api.client.unit.test.ts:195]: I know this is dummy data, but the id should be CCD-######-######

File: test/client/api.client.unit.test.ts (hunk 1 above)
Comment (ethatcher83) [reply test/client/api.client.unit.test.ts]: good spot thanks

File: test/client/api.client.unit.test.ts (hunk 2 above)
Comment (ethatcher83) [reply test/client/api.client.unit.test.ts]: thanks, updated

**Review by nabeelamirch — APPROVED** [review]

**Pull request merged**